sympy/physics/vector/printing.py

```diff
@@ -148,6 +147,10 @@ def _print_Derivative(self, der_expr):
             base = r"\ddot{%s}" % base
         elif dots == 3:
             base = r"\dddot{%s}" % base
+        elif dots == 4:
+            base = r"\ddddot{%s}" % base
+        else: # Fallback to standard printing
+            return LatexPrinter().doprint(der_expr)
```

Comment:
@oscargus, do you remember why you used `LatexPrinter().` and not `super(VectorPrettyPrinter, self)._print_Derivative` like you did for the other lines? (xref gh-19678)